pkg/sql/sem/builtins/builtins.go

```diff
@@ -3597,6 +3598,49 @@ may increase either contention or retry errors, or both.`,
 		},
 	),
 
+	"crdb_internal.payloads_for_span": makeBuiltin(
+		tree.FunctionProperties{Category: categorySystemInfo},
+		tree.Overload{
+			Types:      tree.ArgTypes{{"span ID", types.Int}},
+			ReturnType: tree.FixedReturnType(types.Jsonb),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				builder := json.NewArrayBuilder(len(args))
+				var payloads []string
+				spanID := uint64(*(args[0].(*tree.DInt)))
+
+				var payload json.JSON
```

Comment:
Is there a reason we need to let the scope for `payload` and `err` extend outside the Structured closure below?

Reply:
Not anymore! Thanks, removing locally.